pkgs/by-name/bl/bleep/package.nix

```
@@ -59,6 +60,11 @@ stdenvNoCC.mkDerivation (finalAttrs: {
         --zsh <(bleep install-tab-completions-zsh --stdout) \
     '';
 
+  passthru.tests.version = testers.testVersion {
+    package = finalAttrs.finalPackage;
+    command = "bleep --help | grep \"Bleeping\" | awk '{print $5}' | tr -d ')'";
```

```suggestion
    command = "bleep --help | grep \"Bleeping\" | awk '{print $5}' | tr -d ')'";
    command = "bleep --help | sed -n -e '/Bleeping/s/[^0-9.]//gp'"
```

I guess a single sed call with "show only lines with `Bleeping` and remove all other characters than `0-9.`" is more simple

---

But also, "[versionCheckHook](https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook) should be preferred"
ref. https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion

---

Will this still trigger passthru tests on ofborg? That is the main reason I added it. Without the passthru it hangs for several days on the passthru testing.